lib/flipper/views/feature.erb

```diff
@@ -1,4 +1,9 @@
 <% if RequestStore.store[:flipper_user_email_for_log].present? || Rails.env.development? %>
+
+  <% actor_type = FLIPPER_FEATURE_CONFIG['features'].dig(feature_name, 'actor_type')%>
```

Comment:
Ideally, these would be in some sort of helper, but I'm not sure that it's worth the effort to open that can of worms since the controller for this view is in the gem.